fitbenchmarking/fitbenchmarking
# Issue #209: Merge parsers and update problem sets

#### Description of Work

Fixes #209 

Removes sasview parser and merges functionality into fitbenchmarking parser


#### Testing Instructions

1. Run some of the changed files and check that output is the same as master for various solvers

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
